css/mistica-common.css

```diff
@@ -0,0 +1,165 @@
+}
+.mistica-border-radius-sheet {
+    border-radius: var(--mistica-border-radius-sheet);
+}
+.mistica-border-radius-mediaSmall {
+    border-radius: var(--mistica-border-radius-mediaSmall);
+}
+
+/* text utility classes */
+.mistica-text-1 {
+    font-size: var(--mistica-font-size-1);
+    line-height: var(--mistica-line-height-1);
+    font-weight: var(--mistica-font-weight-1);
+}
+.mistica-text-2 {
+    font-size: var(--mistica-font-size-2);
+    line-height: var(--mistica-line-height-2);
+    font-weight: var(--mistica-font-weight-2);
+}
+.mistica-text-3 {
+    font-size: var(--mistica-font-size-3);
+    line-height: var(--mistica-line-height-3);
+    font-weight: var(--mistica-font-weight-3);
+}
+.mistica-text-4 {
+    font-size: var(--mistica-font-size-4);
+    line-height: var(--mistica-line-height-4);
+    font-weight: var(--mistica-font-weight-4);
+}
+.mistica-text-5 {
+    font-size: var(--mistica-font-size-5);
+    line-height: var(--mistica-line-height-5);
+    font-weight: var(--mistica-font-weight-5);
+}
+.mistica-text-6 {
+    font-size: var(--mistica-font-size-6);
+    line-height: var(--mistica-line-height-6);
+    font-weight: var(--mistica-font-weight-6);
+}
+.mistica-text-7 {
+    font-size: var(--mistica-font-size-7);
+    line-height: var(--mistica-line-height-7);
+    font-weight: var(--mistica-font-weight-7);
+}
+.mistica-text-8 {
+    font-size: var(--mistica-font-size-8);
+    line-height: var(--mistica-line-height-8);
+    font-weight: var(--mistica-font-weight-8);
+}
+.mistica-text-9 {
+    font-size: var(--mistica-font-size-9);
+    line-height: var(--mistica-line-height-9);
+    font-weight: var(--mistica-font-weight-9);
+}
+.mistica-text-10 {
+    font-size: var(--mistica-font-size-10);
+    line-height: var(--mistica-line-height-10);
+    font-weight: var(--mistica-font-weight-10);
+}
+.mistica-text-cardTitle {
+    font-size: var(--mistica-font-size-cardTitle);
+    line-height: var(--mistica-line-height-cardTitle);
+    font-weight: var(--mistica-font-weight-cardTitle);
+}
+.mistica-text-button {
+    font-size: var(--mistica-font-size-button);
+    line-height: var(--mistica-line-height-button);
+    font-weight: var(--mistica-font-weight-button);
+}
+.mistica-text-tabsLabel {
+    font-size: var(--mistica-font-size-tabsLabel);
+    line-height: var(--mistica-line-height-tabsLabel);
+    font-weight: var(--mistica-font-weight-tabsLabel);
+}
+.mistica-text-link {
+    font-size: var(--mistica-font-size-link);
+    line-height: var(--mistica-line-height-link);
+    font-weight: var(--mistica-font-weight-link);
+}
+.mistica-text-title1 {
+    font-size: var(--mistica-font-size-title1);
+    line-height: var(--mistica-line-height-title1);
+    font-weight: var(--mistica-font-weight-title1);
+}
+.mistica-text-title2 {
+    font-size: var(--mistica-font-size-title2);
+    line-height: var(--mistica-line-height-title2);
+    font-weight: var(--mistica-font-weight-title2);
+}
```

Comment:
I wondering if we should add title3 to tokens.json so users can have all title sizes available. WDYT @yceballost

Reply:
Fine by me! and we will need to include also `title4` soon, right?